operations.py

```diff
@@ -0,0 +1,22 @@
+import pandas as pd
+
+
+# Return most used IP address by using pandas mode(), then selecting element [0], which is the address
+def mf_ip(ip_df: pd.DataFrame) -> str:
```

Comment:
you can use Strategy design pattern https://refactoring.guru/design-patterns/strategy and https://refactoring.guru/design-patterns/strategy/python/example. It would make your app fairly easy to be extended with more "commands" driven by application options.

Reply:
Wow, thanks! I'll try to do something with it, hope it don't require introducing classes (don't get me wrong, I love them, but would breach project consistency imho).

Reply:
Probably won't have time to do that. :( I'll need to postpone exercise till next week to properly dig into it.